content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -215,6 +215,11 @@ Use the following commands to install Containerd on your system:
 ### Prerequisites
 
 ```shell
+cat >> /etc/modules <<EOF
+overlay
+br_netfilter
+EOF
+
```

Comment:
this can create duplicates in /etc/modules

would something like this be more safe?
```
cat > /etc/modules-load.d/containerd.conf <<EOF
overlay
br_netfilter
EOF
```

Reply:
I believe duplicates don't actually create a problem other than being listed as duplicates.

However, "cat >" would overwrite any existing configuration inside /etc/modules which means it's less safe. To be absolutely safe and clean, it would have to check for existence of modules in the config before inserting each module. I believe this would do the trick but also harder to read:

```bash
if ! grep -Eq "^overlay$" /etc/modules; then echo overlay >>/etc/modules; fi
if ! grep -Eq "^br_netfilter$" /etc/modules; then echo br_netfilter>>/etc/modules; fi
```